src/funnels/Depositor.sol

```diff
@@ -118,15 +120,17 @@ contract Depositor {
         emit Deny(usr);
     }
 
-    function setLimits(address gem0, address gem1, uint64 hop, uint128 cap0, uint128 cap1) external auth {
+    function setLimits(address gem0, address gem1, uint24 fee, uint32 hop, uint96 cap0, uint96 cap1) external auth {
         require(gem0 < gem1, "Depositor/wrong-gem-order");
-        limits[gem0][gem1] = PairLimit({
-            hop:  hop,
-            zzz:  limits[gem0][gem1].zzz,
+        limits[gem0][gem1][fee] = PairLimit({
```

Comment:
Should the limit really be set for a fee level? Maybe we could leave the choice of the fee level to the facilitator?

Reply:
Hmmm this is what we discussed in chat, I think both options are valid. I tend more to feel choosing the pool is something that is not needed to be decided by the facilitator as it will split liquidity and probably requires more strategical level of decision than a tactical one.

Reply:
Yes it's not so much because I think the facilitator should decide the value of that param, but more that it may be useful for it to be changeable rapidly by the SubDAO, without waiting for an executive, and have the facilitator just be enacting the will of the SubDAO. In general I guess anything that is not rate-limit-related could be made easy to update without an executive unless there is a good reason not to.

Reply:
Yeah I think there are good arguments for both ways, willing to change if preferred. Let's see what @oldchili thinks about this.

Reply:
Yeah I don't have a strong opinion.
Maybe slightly leaning to keep this change just to better support the case of a slow migration from one pool to another.
In that case if the rate limiting is combined it becomes a bit harder to manage (need the facilitator to only use half of `due` for each pool, etc etc..).

Note that if the subdao wants the freedom of deciding which pool out of two to use without an exec it can set a limit for both pools in advance. The downsize for that is that it doubles the risk that the rate limit is there to protect against, but I think that's something manageable.

Reply:
Ok let's keep it as-is. Resolving.